pkgs/desktops/gnome/extensions/g3kb-switch/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, cmake
+, pkg-config
+, glib
+  # , bash-completion
+}:
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-g3kb-switch";
+  version = "1.2";
+  src = fetchFromGitHub {
+    owner = "lyokha";
+    repo = "g3kb-switch";
+    rev = "1.2";
+    sha256 = "sha256-Nq2psfy51hZQEm57PUSvs17Pp2z6pe30h9BOLbOYV4k=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    glib
+    # bash-completion
+  ];
+  patchPhase = ''
```

Comment:
Overriding `patchPhase` breaks `patches` field. Use `prePatch` or `postPatch` instead. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/patch-phase.md> for more info.